CRAN issue

Please see the problems shown on
<https://cran.r-project.org/web/checks/check_results_lbfgsb3c.html>.

Please correct before 2023-12-11 to safely retain your package on CRAN.

```
    lbfgsb3x.cpp:139:21: warning: too many arguments for format [-Wformat-extra-args]
```

--- a/DESCRIPTION
+++ b/DESCRIPTION
@@ -2,23 +2,23 @@ Package: lbfgsb3c
 Type: Package
 Title: Limited Memory BFGS Minimizer with Bounds on Parameters with
         optim() 'C' Interface
-Version: 2020-3.2
+Version: 2020-3.3
 Authors@R: c(person(given = c("Matthew", "L"), family = "Fidler", role =
               c("aut", "cre"), email = "matthew.fidler@gmail.com"),
 	      person(given = c("John", "C"), family = "Nash", role =
               c("aut"), email = "nashjc@uottawa.ca"),
-              person(given = "Ciyou", family = "Zhu", role = "aut"), 
-              person(given = "Richard", family = "Byrd", role = "aut"), 
+              person(given = "Ciyou", family = "Zhu", role = "aut"),
+              person(given = "Richard", family = "Byrd", role = "aut"),
               person(given = "Jorge", family = "Nocedal", role = "aut"),
               person(given = c("Jose", "Luis"), family = "Morales", role = "aut"))
 Maintainer: Matthew L Fidler <matthew.fidler@gmail.com>
-Description: Interfacing to Nocedal et al. L-BFGS-B.3.0 
+Description: Interfacing to Nocedal et al. L-BFGS-B.3.0
         (See <http://users.iems.northwestern.edu/~nocedal/lbfgsb.html>)
-        limited memory BFGS minimizer with bounds on parameters. 
-        This is a fork of 'lbfgsb3'. 
-	This registers a 'R' compatible 'C' interface to L-BFGS-B.3.0 that uses the same 
+        limited memory BFGS minimizer with bounds on parameters.
+        This is a fork of 'lbfgsb3'.
+	This registers a 'R' compatible 'C' interface to L-BFGS-B.3.0 that uses the same
 	function types and optimization as the optim() function (see writing 'R' extensions
-        and source for details).  This package also adds more stopping criteria as well 
+        and source for details).  This package also adds more stopping criteria as well
         as allowing the adjustment of more tolerances.
 License: GPL-2
 Depends: R (>= 3.6)
@@ -28,7 +28,7 @@ LinkingTo: Rcpp (>= 0.12.3), RcppArmadillo
 Encoding: UTF-8
 VignetteBuilder: knitr
 NeedsCompilation: yes
-RoxygenNote: 7.0.2
+RoxygenNote: 7.2.3
 Author: Matthew L Fidler [aut, cre],
   John C Nash [aut],
   Ciyou Zhu [aut],

--- a/NEWS.md
+++ b/NEWS.md
@@ -1,10 +1,12 @@
-# lbfgsb3c development changes
+# lbfgsb3c 2020-3.3 changes
 
 * Now allow `rho=NULL` to work the same as if `rho` was not supplied
 
 * Be More careful about `$convergence` by adding a default value of `NA_INTEGER`
 
 * `$convergence` is now an integer instead of a real number
+
+* Fix too many arguments for format as requested by CRAN
 
 * Added a `NEWS.md` file to track changes to the package.
 

--- a/NEWS.md
+++ b/NEWS.md
@@ -10,7 +10,7 @@
 
 * Added a `NEWS.md` file to track changes to the package.
 
-# News before lbfgsb3c
+# lbfgsb3c 2020-3.2 prior changes and information
 
 To do
 

--- a/src/lbfgsb3x.cpp
+++ b/src/lbfgsb3x.cpp
@@ -136,7 +136,7 @@ extern "C" void lbfgsb3C_(int n, int lmm, double *x, double *lower,
       	  itask2=27;
       	  itask=3; // Stop -- gives the right results and restores gradients
 	  if (trace > 2){
-	    Rprintf("CONVERGENCE: Parameters differences below xtol.\n", maxit);
+	    Rprintf("CONVERGENCE: Parameters differences below xtol.\n");
 	  }
 	  doExit=1;
       	}
@@ -263,7 +263,7 @@ Rcpp::List lbfgsb3cpp(NumericVector par, Function fn, Function gr, NumericVector
 		  1 if x(i) has only a lower bound,
 		  2 if x(i) has both lower and upper bounds,
 		  3 if x(i) has only an upper bound.
-    */    
+    */
     nbd[i] = 0;
     if (R_FINITE(low[i])) nbd[i] = 1;
     if (R_FINITE(up[i]))  nbd[i] = 3 - nbd[i];
@@ -346,7 +346,7 @@ Rcpp::List lbfgsb3cpp(NumericVector par, Function fn, Function gr, NumericVector
   taskList[24]="WARNING: STP .eq. STPMIN"; // 25
   taskList[25]="WARNING: XTOL TEST SATISFIED"; //
   taskList[26] = "CONVERGENCE: Parameters differences below xtol";
-  taskList[27] = "Maximum number of iterations reached";      
+  taskList[27] = "Maximum number of iterations reached";
 
   ret["message"]= CharacterVector::create(taskList[fail-1]);
   if (addInfo) ret["info"] = lbfgsb3Cinfo;

